store/src/java-test/com/zimbra/cs/service/util/VolumeConfigUtilTest.java

```diff
@@ -169,13 +178,15 @@ public void testParseModifyVolumeInplaceUpgradeRequest() throws ServiceException
         JSONObject properties = new JSONObject();
         properties.put(AdminConstants.A_VOLUME_STORAGE_TYPE, AdminConstants.A_VOLUME_S3);
         properties = externalVolumeInfoHandler.readServerProperties(volumeId);
```

Comment:
the same line is added at line no185, please remove one of them

Reply:
done